internal/handlers/ping.go

```diff
@@ -0,0 +1,24 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/arseniy96/url-shortener/internal/logger"
+)
+
+func (s *Server) Ping(writer http.ResponseWriter, request *http.Request) {
+	if s.storage.GetMode() != 2 {
```

Comment:
```suggestion
	if s.storage.GetMode() != Something {
```
Магические числа лучше убирать в константы, и лучше чтобы внутри была понятная строка, а не число